src/main/java/patio/group/domain/Group.java

```diff
@@ -64,6 +64,9 @@
   @OneToMany(mappedBy = "group")
   private Set<UserGroup> users;
 
+  @Column(name = "voting_duration")
+  private Integer votingDuration; /* in hours */
```

Comment:
Are you sure you only want hours ? This avoid setting minutes

Reply:
I'll double check it wiht Miryam...

Reply:
She has confirm that we don't intent to manage minutes, but it worth it to give her the chance. :+1: